### PR TITLE
Au qa 72 guard order mix test keyword replacement

### PR DESCRIPTION
**Pull request by GreeshmaPK**

### CODE REVIEW

File: Testcases/OverrideTests/override1.robot [hunk 1]
```diff
@@ -113,12 +114,13 @@ SFCMinMaxCheck
     apiresources.verifyBOPOriginOfListedAHUsAreControl  ${test_input}[ahu_1]  ${test_input}[ahu_2]  ${test_input}[ahu_3]
     #55)Verify On/Off Value = ON for AHUs CAC_10, CAC_13, and CAC_15
     apiresources.verifyBOPValueOfListedAHUsAreON  ${test_input}[ahu_1]  ${test_input}[ahu_2]  ${test_input}[ahu_3]
-    #56)Wait 5 minutes
+    #56)Wait 5 minutes <Strike through in Confluence>
```
Comment (ananiac) [inline Testcases/OverrideTests/override1.robot:117]: update Strikethrough in Confluence to
Post the decision on 9 Nov 2021 meeting the step is been removed from testcases and Strikethrough  in Confluence

File: Testcases/OverrideTests/override1.robot [hunk 2]
```diff
@@ -113,12 +114,13 @@ SFCMinMaxCheck
...
 #    common.waitForMinutes    5
-    #57)Verify Supply Fan Value = 72.0% for AHUs CAC_10, CAC_13, and CAC_15
+    #57)Verify Supply Fan Value = 72.0% for AHUs CAC_10, CAC_13, and CAC_15 <Strike through in Confluence>
```
Comment (ananiac) [inline Testcases/OverrideTests/override1.robot:119]: update Strikethrough in Confluence to
Post the decision on 9 Nov 2021 meeting the step is been removed from testcases and Strikethrough  in Confluence

File: Testcases/OverrideTests/override1.robot (hunk 1 above)
Comment (GreeshmaPK) [reply Testcases/OverrideTests/override1.robot:117]: done

File: Testcases/OverrideTests/override1.robot (hunk 2 above)
Comment (GreeshmaPK) [reply Testcases/OverrideTests/override1.robot:119]: done

**Pull request merged**